src/papi_events.csv

```diff
@@ -1954,6 +1954,7 @@ PRESET,PAPI_L1_ICM,NOT_DERIVED,IFETCH_MISS
 #
 CPU,arm_ac15
 CPU,arm_ac57
```

Comment:
You can put this change in another commit or, even, a different PR. The first part of this patch does not need libpfm4 support and can be merged immediately. Moreover, there are no preset events for `arm_ac76` yet, so unless you have a list of well tested preset events you would like to suggest, I would not make any changes to this file.